src/profile.ts

```diff
@@ -461,6 +465,7 @@ export default class Profile {
 ; cooling = ${this.cooling.map(boolToIntString).join(',')}
 ; cooling_tube_length = ${this.coolingTubeLength}
 ; cooling_tube_retraction = ${this.coolingTubeRetraction}
+; cooling_module_speed = ${this.coolingModuleSpeed.join(',')}
```

Comment:
This isn't a valid setting so it shouldn't be included in the profile file itself